src/components/common/navigation/NavigationRoutes.tsx

```diff
@@ -297,46 +304,46 @@ export default function NavigationRoutes() {
                                                     loginCount={loginCount}
                                                 />
                                             )}
-                                        />
+                                        />}
                                         <Route
                                             path={`${URLS.RESOURCE_BROWSER}/:clusterId?/:namespace?/:nodeType?/:node?`}
                                         >
                                             <ResourceBrowserContainer />
                                         </Route>
-                                        <Route path={URLS.CHARTS} render={() => <Charts />} />
-                                        <Route
+                                        {!window._env_.K8S_CLIENT && <Route path={URLS.CHARTS} render={() => <Charts />} />}
+                                        {!window._env_.K8S_CLIENT && <Route
```

Comment:
same here

Reply:
Or we can switch set of routes based on `window._env_.K8S_CLIENT`?

Reply:
done